standards/WIT-SVID.md

```diff
@@ -0,0 +1,167 @@
+# The WIT SPIFFE Verifiable Identity Document
+
+> [!WARNING]  
+> This version of the document belongs to an experimental branch of the SPIFFE standards that explores integration with the WIMSE standards. Contents of this document are subject to breaking change and should not be considered stable.
+>
+> The current stable version of this document can be found at https://github.com/spiffe/spiffe/blob/main/standards/SPIFFE_Workload_Endpoint.md
+
+## Status of this Memo
+
+This document specifies an identity document standard for the internet community, and requests discussion and suggestions for improvements. Distribution of this document is unlimited.
+
+## Table of Contents
+
+## 1. Introduction
+
+The Workload Identity Token (WIT) is a token format specified by the IETF WIMSE working group in the [WIMSE Workload Credentials][1] document. The WIT binds a public key to the identity of a workload.
+
+The WIT-SVID is a sub-profiling of this token format for use in SPIFFE contexts. This document will not redefine key elements specified within the upstream [WIMSE Workload Credentials][1] document and as such the reader should be familiar with this document and its contents before implementing WIT-SVID.
+
+WIT-SVIDs are JSON Web Signature (JWS) data structures.
+
+### 2. JOSE Header
+
+This section describes the JOSE headers that are defined for the WIT-SVID.
```

Comment:
Just had a look at the standard and "header" means the entire "JOSE header", i.e., the JSON object.

We should use "header parameter" for the key-value pairs in there.